modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProducts.testcase

```diff
@@ -297,12 +345,30 @@ definition {
 	}
 
 	@description = "This is a test for COMMERCE-5803. UseSortingByNameAndDate"
-	@ignore = "true"
 	@priority = "3"
 	test UseSortingByNameAndDate {
 		property portal.acceptance = "false";
 
-		// TODO COMMERCE-5803 UseSortingByNameAndDate
+		CommerceAccelerators.initializeNewSiteViaAccelerator(siteName = "Minium");
+
+		CommerceNavigator.gotoPortlet(
+			category = "Product Management",
+			portlet = "Products");
+		CommerceNavigator.sortField(sortFieldName = "Name");
+
+		CommerceNavigator.viewFilterResultsSpecificOrder(resultsList = "ABS Sensor,Ball Joints,Bearings");
```

Comment:
Here is where resultsOrder is being utilized. By having that variable, you can use it as a constant so that when asserting for text you can make sure it asserts for "ABS Sensor" then "Ball Joints" etc.